### DISCUSSION TITLE
So lets discuss this access to Codespaces

### DISCUSSION THREAD

**Discussion by lythix** [mishmanners/GitHub-Like-A-Boss#4] [General] [answered]
Still waiting. ;)

**Comment by mishmanners** [comment, marked as answer]
APPLY HERE: https://github.com/features/codespaces/

First round of beta users will be released in two weeks!

**Reply by lythix** [reply]
I don't like the answer tho...

**Reply by lythix** [reply]
TWO WEEKS??!
